packages/page-accounts/src/Accounts/modals/IdentityMain.tsx

```diff
@@ -27,7 +27,7 @@ interface WrapProps {
 }
 
 interface ValueState {
-  info: Record<string, unknown>;
+  info: Record<string, any>;
```

Comment:
Just a question around this one - where was it needed? (Just interested since the best-practice is to use `unknown` (and then cast as required), but alas, there is still a lot of work left, just been touching them as it comes into focus.

Reply:
It's not needed anymore, reverting. The problem was when I did something like:
```js
Object.values(info).forEach((i: Record<string, string>) => console.log(i.bla));
```
where it was complaining on the `i`:
> Argument of type '(i: Record<string, string>) => void' is not assignable to parameter of type '(value: unknown, index: number, array: unknown[]) => void'.